kornia/contrib/extract_patches.py

```diff
@@ -125,20 +126,25 @@ class CombineTensorPatches(nn.Module):
 
     def __init__(
         self,
+        orig_size: Union[int, Tuple[int, int]],
         window_size: Union[int, Tuple[int, int]],
         unpadding: Union[int, Tuple[int, int]] = 0,
     ) -> None:
         super().__init__()
+        self.orig_size: Tuple[int, int] = _pair(orig_size)
         self.window_size: Tuple[int, int] = _pair(window_size)
         pad: Tuple[int, int] = _pair(unpadding)
         self.unpadding: Tuple[int, int, int, int] = (pad[0], pad[0], pad[1], pad[1])
 
     def forward(self, input: torch.Tensor) -> torch.Tensor:  # type: ignore
-        return combine_tensor_patches(input, self.window_size, stride=self.window_size, unpadding=self.unpadding)
+        return combine_tensor_patches(
+            input, self.orig_size, self.window_size, stride=self.window_size, unpadding=self.unpadding
+        )
 
 
 def combine_tensor_patches(
     patches: torch.Tensor,
+    orig_size: Tuple[int, int] = (16, 16),
```

Comment:
Since the function wasn't working correctly before, is there a need to maintain backwards compatibility?

Reply:
While on the topic, wouldn't it better to remove the default arguments for combine and extract?  While using these a user should explicitly designate the window_size, stride and original_size (for combine) parameters so that they get a reasonable result. Otherwise we run the risk of them specifying one parameter and getting an incorrect result or an OOM since default stride is 1.